hazardous/survival_mixin.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+
+
+class SurvivalMixin:
+    def predict_cumulative_hazard_function(self, X_test, times):
+        survival_probs = self.predict_survival_function(X_test, times)
+        cumulative_hazards = -np.log(survival_probs + 1e-8)
+        return cumulative_hazards
```

Comment:
I would rather not introduce a function that method that derives a cumulative hazard from a survival function estimate as this cannot generalize to the competing risks setting. Let's remove this mixin.

Reply:
yes indeed, let's remove it